API/Services/ArchiveService.cs

```diff
@@ -356,7 +355,8 @@ private static bool ValidComicInfoArchiveEntry(string fullName, string name)
                 {
                     using var archive = ZipFile.OpenRead(archivePath);
 
-                    var entry = archive.Entries.FirstOrDefault(x => ValidComicInfoArchiveEntry(x.FullName, x.Name));
+                    var entry = archive.Entries.FirstOrDefault(x => x.FullName == ComicInfoFilename) ??
```

Comment:
From what I've seen, FullName might not always be populated, hence why I had:
(string.IsNullOrEmpty(fullName) && name.Equals(ComicInfoFilename)).

Reply:
Do you have an example I can use to reproduce that? I'd like to understand and get something like it for the unit tests. I had assumed that this was meant for rar archives where Path.GetDirectoryName would return null (which now that I have read on that method is indeed unlikely).

Reply:
The real concern here is that when FullName is absent, we cannot really tell if ComicInfo is in the root directory of the archive. This needs to be understood.